### ISSUE TITLE
Add write element variable names and values

### ISSUE THREAD

**Issue by lxmota** [cmhamel/Exodus.jl#24]
These are missing in ElementVariables.jl and they are needed for output of stress, strain and other variables that define the state of a material. We are using the equivalent Python ones here https://github.com/lxmota/norma/blob/main/src/time_integrator.jl#L220 and here https://github.com/lxmota/norma/blob/main/src/time_integrator.jl#L378, but we woud like to transition to Exodus.jl

**Comment by cmhamel** [comment]
@lxmota I just added these capabilities. Thanks for the nudge. The methods you want have the following pattern

read_element_variable_*

Also, if you started using the package note that I tweaked a few things for quality of life. Mainly coordinates are now a n_nodes x n_dims arrays rather than the transpose and the block data structure now stores the connectivity for a block as n_els x n_nodes_per_el rather than the long list exodus natively stores.

I'm about to release these updates so they should be ready to use in an hour or so.
